Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/InputActionAssetDrawer.cs

```diff
@@ -0,0 +1,71 @@
+#if UNITY_EDITOR && UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+
+using UnityEditor;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    [CustomPropertyDrawer(typeof(InputActionAsset))]
+    internal class InputActionAssetDrawer : PropertyDrawer
+    {
+        static readonly string[] k_ActionsTypeOptions = new[] { "Project-Wide Actions", "Actions Asset" };
+
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            EditorGUI.BeginProperty(position, label, property);
+
+            var isAssetProjectWideActions = IsAssetProjectWideActions(property);
+            var selectedAssetOptionIndex = isAssetProjectWideActions ? 0 : 1;
```

Comment:
Would be tidier if this was an enum rather than 0/1 int value

You can use EditorGUILayout.EnumPopup below to assist this

Reply:
Added those changes now